src/client/visualizations/bar-chart/bar-chart.tsx

```diff
@@ -94,8 +92,7 @@ function getFilterFromDatum(splits: Splits, dataPath: Datum[]): Filter {
         case SplitType.string:
```

Comment:
What about SplitType.boolean? Or SplitType for boolean is a string?

Reply:
https://github.com/allegro/turnilo/issues/563 :)

Reply:
commented in the issue